### PR TITLE
Symlink data folder to bin folder with VS install scirpt

### PR DESCRIPTION
**Pull request by marcovmun**
Today I couldn't run the develop branch because of the langue file chances. This add a build event to Visual Studio that always copy data map to the bin/data map. Then you have always the newest data files when running the game. I seemed to me very handy.
What is your opinion on this subject?

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
This was in before and it was removed for some reason.

**Comment by janisozaur** [comment]
I mentioned this the other day on gitter – I have my `data/` symlinked, so it travels with my git checkout and I haven't once bothered with mismatched files or any of such kind. Why won't you symlink the `data/` as well?

**Comment by marcovmun** [comment]
Before today I didn't now that my data folder was not been updated. But symlink the folder is also a option. Maybe it is a option to modify the build.ps1 script to symlink the data folder to bin\data if it is possible and otherwise copy the files

**Comment by IntelOrca** [comment]
Yes I was thinking of updating the script to create a symlink. The reason I removed it was because it was unnecessary to copy the files 99% of the time as it would do it every run or debug. There was also another issue where msbuild doesn't always seem to copy the files unless you do a rebuild or something. So in the end I just made it an explicit action that you do in PowerShell. Running ```build all``` like it states in the readme should copy the data files.

**Comment by duncanspumpkin** [comment]
There is also the issue of g2 not getting rebuilt

**Comment by marcovmun** [comment]
Oké, that are logical reasons. Then this pull request can be closed or should it be stay open for the discussion?
Updating the script would be nice.

**Comment by IntelOrca** [comment]
@duncanspumpkin oh yeah, that was the more annoying thing that kept happening every run.

File: scripts/ps/build.ps1 [hunk 1]
```diff
@@ -28,9 +28,24 @@ $openrct2Path = Join-Path $binPath "openrct2.exe"
 
 function Build-Data()
 {
-    Write-Host "Copying data to bin..." -ForegroundColor Cyan
-    New-Item -Force -ItemType Directory $binPath > $null
-    Copy-Item -Force -Recurse "$rootPath\data" $binPath
+	if(Test-Path "$binPath\data")
```
Comment (IntelOrca) [inline scripts/ps/build.ps1]: Please use spaces, not tabs for script files.

File: scripts/ps/install.ps1 [hunk 2]
```diff
@@ -19,6 +19,7 @@ $libsVersion = 7
 # Get paths
 $rootPath        = Get-RootPath
 $libsPath        = Join-Path $rootPath "lib"
+$binPath        = Join-Path $rootPath "bin"
```
Comment (IntelOrca) [inline scripts/ps/install.ps1]: Please align the equal signs up.

File: scripts/ps/install.ps1 [hunk 3]
```diff
@@ -34,6 +35,32 @@ if ($currentLibsVersion -ge $libsVersion)
     $updateLibs = $false
 }
 
+#symlink data to bin\data
+try {
```
Comment (IntelOrca) [inline scripts/ps/install.ps1]: Braces go on a line of their own.

File: scripts/ps/build.ps1 [hunk 4]
```diff
@@ -28,9 +28,24 @@ $openrct2Path = Join-Path $binPath "openrct2.exe"
...
+	{
+		$object = Get-Item "$binPath\data"
+		
+		if(-not ([bool]($object.Attributes -band [IO.FileAttributes]::ReparsePoint)))
+		{
+			Write-Host "Copying data to bin..." -ForegroundColor Cyan
+			New-Item -Force -ItemType Directory $binPath > $null
+			Copy-Item -Force -Recurse "$rootPath\data" $binPath
+		}
+		else{
```
Comment (IntelOrca) [inline scripts/ps/build.ps1]: Braces go on a line of their own

File: scripts/ps/install.ps1 [hunk 5]
```diff
@@ -34,6 +35,32 @@ if ($currentLibsVersion -ge $libsVersion)
...
+} 
+catch [System.Management.Automation.ParameterBindingException] {
+	Write-Host "Your powershell can not create symlinks" -ForegroundColor Red
+	Write-Host "Copying data to bin..." -ForegroundColor Cyan
+	New-Item -Force -ItemType Directory $binPath > $null
+	Copy-Item -Force -Recurse "$rootPath\data" $binPath
+} 
+catch {
+    Write-Host "Symlink not possible" -ForegroundColor Red
+    if($force) {
```
Comment (IntelOrca) [inline scripts/ps/install.ps1]: Please add a space between ```if``` and ```(```

**Comment by IntelOrca** [comment]
@marcovmun the logic looks fine, can you just conform the formatting like the rest of the file and then we should be able to merge it.

**Comment by marcovmun** [comment]
I have chance it so it conform the formatting of the rest of the file.

**Pull request closed without merging**